src/main/java/TaskManager.java

```diff
@@ -3,14 +3,14 @@
 
 public class TaskManager {
     /**
-     * list to store all reminders
+     * list to store all tasks
      */
```

Comment:
Good use of comments to explain each line of code

Reply:
Thanks!